reporting/fetch-reports.ts

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env node -r esm -r ts-node/register
```

Comment:
**Note:** It's probably worth diffing this file against the previous file to see what has actually changed, rather than attempting to view/review it in its entirety.

I believe something like the following (while on this branch) should work:

```
git diff staging:reporting/fetch-reports.js HEAD:reporting/fetch-reports.ts
```